src/test/java/pl/antonina/tasks/parent/ParentServiceTest.java

```diff
@@ -35,35 +36,37 @@
 
     @BeforeEach
     void beforeEach() {
-        parentService = new ParentService(parentRepository, parentMapper, userRepository, userService);
+        parentService = new ParentService(parentRepository, parentMapper, userRepository, userService, loggedUserService);
     }
 
     @Test
     void getParent() {
-        long id = 123;
-
         Parent parent = new Parent();
         ParentView parentView = new ParentView();
 
-        when(parentRepository.findById(id)).thenReturn(Optional.of(parent));
+        Principal principal = mock(Principal.class);
+        when(loggedUserService.getParent(principal)).thenReturn(parent);
         when(parentMapper.mapParentView(parent)).thenReturn(parentView);
 
-        ParentView parentViewResult = parentService.getParent(id);
+        ParentView parentViewResult = parentService.getParent(principal);
+
         assertThat(parentViewResult).isEqualTo(parentView);
     }
 
     @Test
     void addParent() {
         Gender gender = Gender.FEMALE;
         String name = "Antonina";
+        UserType userType = UserType.PARENT;
```

Comment:
Przydałyby się finale, bo skoro potem jest porównanie do tych zmiennych, to powinniśmy być pewni, że ich wartości się nie zmieniły (ta sama uwaga do dalszych testów).

Reply:
ok :) sporo będzie final'ów :)